Tella/Data/Networking/Repositories/UwaziServerRepositories.swift

```diff
@@ -13,9 +13,9 @@ struct UwaziServerRepository: WebRepository {
 
     func login(username: String,
                password: String,
-               serverURL: String) -> AnyPublisher<(UwaziLoginResult,HTTPURLResponse?), APIError> {
+               serverURL: String) -> AnyPublisher<(BoolResponse,HTTPURLResponse?), APIError> {
 
-        let call : AnyPublisher<(UwaziLoginResult,HTTPURLResponse?), APIError> = callReturnsHeaders(endpoint: API.login((username: username, password: password, serverURL: serverURL)))
+        let call : AnyPublisher<(BoolResponse,HTTPURLResponse?), APIError> = callReturnsHeaders(endpoint: API.login((username: username, password: password, serverURL: serverURL)))
```

Comment:
@rob-horizontal-iOS I've made the change and I pushed to develop, so could you rebase uwazi feature onto develop and get the updated function getAPIResponse and apply it

Reply:
@dhekra-rouatbi  I have updated as per your instruction

Reply:
Thanks @rob-horizontal-iOS